app/experimenter/base/__init__.py

```diff
@@ -36,3 +40,10 @@ def get_uploads_storage():
         )
 
     return FileSystemStorage()
+
+
+# Borrowing Django's DefaultStorage lazy object approach
+# https://github.com/django/django/blob/ddae36700d1c07cbca15a6821bd99d480294c137/django/core/files/storage.py#L271-L273
+class UploadsStorage(LazyObject):
+    def _setup(self):
+        self._wrapped = get_uploads_storage()
```

Comment:
Made some tweaks that *should* make overriding upload file storage easier on a test-by-test level. Still thinking about how to do it without having to decorate a lot of tests or make changes to otherwise unaffected tests